docs/integrations/system_metrics.md

```diff
@@ -1,28 +1,68 @@
-By default, **Logfire** does not collect system metrics.
+The [`logfire.instrument_system_metrics()`][logfire.Logfire.instrument_system_metrics] method can be used to collect system metrics with **Logfire**, such as CPU and memory usage.
 
-To enable metrics, you need to install the `logfire[system-metrics]` extra:
+## Installation
+
+Install `logfire` with the `system-metrics` extra:
 
 {{ install_logfire(extras=['system-metrics']) }}
 
-### Available Metrics
-
-Logfire collects the following system metrics:
-
-* `system.cpu.time`: CPU time spent in different modes.
-* `system.cpu.utilization`: CPU utilization in different modes.
-* `system.memory.usage`: Memory usage.
-* `system.memory.utilization`: Memory utilization in different modes.
-* `system.swap.usage`: Swap usage.
-* `system.swap.utilization`: Swap utilization
-* `system.disk.io`: Disk I/O operations (read/write).
-* `system.disk.operations`: Disk operations (read/write).
-* `system.disk.time`: Disk time (read/write).
-* `system.network.dropped.packets`: Dropped packets (transmit/receive).
-* `system.network.packets`: Packets (transmit/receive).
-* `system.network.errors`: Network errors (transmit/receive).
-* `system.network.io`: Network I/O (transmit/receive).
-* `system.network.connections`: Network connections (family/type).
-* `system.thread_count`: Thread count.
-* `process.runtime.memory`: Process memory usage.
-* `process.runtime.cpu.time`: Process CPU time.
-* `process.runtime.gc_count`: Process garbage collection count.
+## Usage
+
+```py
+import logfire
+
+logfire.configure()
+
+logfire.instrument_system_metrics()
+```
+
+Then in your project, click on 'Dashboards' in the top bar, click 'New Dashboard', and select 'Basic System Metrics' from the dropdown.
+
+## Configuration
+
+By default, `instrument_system_metrics` collects only the metrics it needs to display the 'Basic System Metrics' dashboard. You can choose exactly which metrics to collect and how much data to collect about each metric. The default is equivalent to this:
+
+```py
+logfire.instrument_system_metrics({
+    'system.cpu.simple_utilization': None,
```

Comment:
This is a new metric I will implement. It's similar to `system.cpu.utilization` but without any attributes, meaning just one data point per export instead of one data point per CPU core per CPU state. It'll just be the sum of 'busy' states and the average over all cores.

In the future we might want to add similar metrics which have a single attribute, e.g. `simple_utilization_by_cpu` and `simple_utilization_by_mode`, to get more detail without multiplying the number of rows. Not a priority right now.

I'm a little hesitant to create new metric names under the official `system.cpu.` namespace, but it seems probably OK. My impression from https://opentelemetry.io/docs/specs/semconv/general/metrics/ and https://opentelemetry.io/docs/specs/semconv/system/system-metrics/ was that this was better than adding a `logfire.` prefix.

Reply:
Can you add a pop over explaining what `None` means or linking to the description below.